docs/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## v4.1.1 (2022-05-31)
 - fix: failed to build debugtalk.go without go.mod
 - fix: avoid to escape from html special characters like '&'
+- fix: add ref step name info in html report
+- fix: incorrect to regenerate .debugtalk_gen.py
```

Comment:
fix: failed to regenerate debugtalk_gen.go and .debugtalk_gen.py correctly

Reply:
got it